deploy/scripts/k8s-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2021 CYBERCRYPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+source ./scripts/encryptonize_env
+source ./scripts/gcp_env
+
+cat ../kubernetes/encryption-service/encryptonize-service.yaml | envsubst | kubectl apply -f -
+cat ../kubernetes/encryption-service/encryptonize-ingress.yaml | envsubst | kubectl apply -f -
```

Comment:
No reason to call cat:
```suggestion
envsubst < ../kubernetes/encryption-service/encryptonize-service.yaml | kubectl apply -f -
envsubst < ../kubernetes/encryption-service/encryptonize-ingress.yaml | kubectl apply -f -
```